packages/UI/examples/server-example/src/server.ts

```diff
@@ -53,11 +53,9 @@ app.post("/api/verify", async (req: Request, res: Response) => {
     if (payload.pdfUrl) {
       const response = await axios.get(payload.pdfUrl, { responseType: "arraybuffer" });
       content = Uint8Array.from(Buffer.from(response.data, "base64"));
-    } else {
-      content = Uint8Array.from(Buffer.from(payload.content || "", "base64"));
+      payload.content = content.toString();
     }
 
-    payload.content = content.toString();
```

Comment:
I think we don't need this line, converting the string to string doesn't make any sense, maybe you are confused we sometimes read the content as message bytes and sometimes we read it as a string

```ts
      payload.content = content.toString();
```